agent/utils/ephemeral_ports_test.go

```diff
@@ -196,7 +196,8 @@ func TestGetHostPort(t *testing.T) {
 
 	for _, tc := range testCases {
 		if tc.resetLastAssignedHostPort {
-			tracker.SetLastAssignedHostPort(0)
+			// need to reset the tracker to avoid getting data from previous test cases
+			ResetTracker()
```

Comment:
This function `ResetTracker()` has been reviewed in https://github.com/aws/amazon-ecs-agent/pull/3585.